src/test/univalue_tests.cpp

```diff
@@ -49,7 +49,7 @@ BOOST_AUTO_TEST_CASE(univalue_constructor)
 
     double vd = -7.21;
     UniValue v7(vd);
-    BOOST_CHECK(v7.isNum());
+    BOOST_CHECK(v7.isReal());
```

Comment:
Because now there is a distinction between int and real (to make it JSON Spirit compatible), things like -7.21 are VREAL and not VNUM. I decided to make VREAL a standalone type instead of a VNUM subtype.